### PR TITLE
chore: fixed capitalization on action toggle readme and minor edits

### PR DESCRIPTION
**Pull request by Falkicon**
# Description

Fixed capitalization on action toggle readme and made minor updates to the description.

## Issue type checklist

- [x] **Chore**: A change that does not impact distributed packages.
- [ ] **Bug fix**: A change that fixes an issue, link to the issue above.
- [ ] **New feature**: A change that adds functionality.

## Process & policy checklist

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [x] I have updated the project documentation to reflect my changes.
- [x] I have read the [CONTRIBUTING](https://github.com/Microsoft/fast-dna/blob/master/CONTRIBUTING.md) documentation and followed the [standards](https://microsoft.github.io/fast-dna/docs/en/contributing/standards) for this project.

### CODE REVIEW

File: packages/fast-components-react-msft/src/action-toggle/README.md [hunk 1]
```diff
@@ -1,12 +1,12 @@
-# Action Toggle
+# Action toggle
 
-*Action toggle* should be used as a control on or around an interactive region of the page that affects the region or items within the region. It usually represents a switch between one of two states.
+Use *action toggle* to switch between two on screen binary states.
```
Comment (nalogiudice) [inline packages/fast-components-react-msft/src/action-toggle/README.md]: I think `on screen` should be `on-screen`

File: packages/fast-components-react-msft/src/action-toggle/README.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 
-The "selectedGlyph" and "unselectedGlyph" props take a function that returns a ReactNode to render the glyph (ie. a [render prop](https://reactjs.org/docs/render-props.html)).  Note that the function should accept a string parameter that is applied as a class name to the top level element of the resulting node.  This class name will be populated by the *action toggle* component to use an internally generated class ("actionToggle_selectedGlyph" & "actionToggle_unselectedGlyph") to apply to the glyphs for styling. 
+The "selectedGlyph" and "unselectedGlyph" props take a function that returns a ReactNode to render the glyph (ie. a [render prop](https://reactjs.org/docs/render-props.html)).  Note that the function should accept a string parameter that is applied as a class name to the top level element of the resulting node.  This class name will be populated by the *action toggle* component to use an internally generated class ("actionToggle_selectedGlyph" & "actionToggle_unselectedGlyph") to apply to the glyphs for styling.
```
Comment (nalogiudice) [inline packages/fast-components-react-msft/src/action-toggle/README.md]: nit: `i.e.`
Also are we normalizing the spacing at the beginning of sentences to be only one space? This has two.

File: packages/fast-components-react-msft/src/action-toggle/README.md (hunk 1 above)
Comment (janechu) [reply packages/fast-components-react-msft/src/action-toggle/README.md]: Or maybe just use "visible"

File: packages/fast-components-react-msft/src/action-toggle/README.md (hunk 2 above)
Comment (Falkicon) [reply packages/fast-components-react-msft/src/action-toggle/README.md]: Yes, spacing should be 1, fixed.

File: packages/fast-components-react-msft/src/action-toggle/README.md (hunk 1 above)
Comment (Falkicon) [reply packages/fast-components-react-msft/src/action-toggle/README.md]: I went with on-screen to avoid confusion with the visible state.

**Pull request merged**